src/batch/src/executor/aggregation/orderby.rs

```diff
@@ -83,42 +82,57 @@ impl Aggregator for ProjectionOrderBy {
 
     async fn update_multi(
         &mut self,
-        input: &DataChunk,
+        input: &StreamChunk,
         start_row_id: usize,
         end_row_id: usize,
     ) -> Result<()> {
         self.unordered_values.reserve(end_row_id - start_row_id);
         for row_id in start_row_id..end_row_id {
-            let (row, vis) = input.row_at(row_id);
+            let (op, row, vis) = input.row_at(row_id);
+            assert_eq!(op, Op::Insert, "only support append");
             if vis {
                 self.push_row(row)?;
             }
         }
         Ok(())
     }
```

Comment:
What about adding another method for stream chunk? Like, `update_stream` or `update_retractable`, so that batch-specific aggregators can just `unimplemented` it. This also helps optimize `update_multi` performance if possible. Also, seeing `StreamChunk` in batch seems a little bit weird.